firebase-firestore/CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 - [fixed] Fixed an internal assertion that was triggered when an
   update with a `FieldValue.serverTimestamp()` and an update with a
   `FieldValue.increment()` were pending for the same document (#491).
+- [changed] Improved performance for queries with filters that only return a
+  small subset of the documents in a collection.
```

Comment:
I'm not sure this is a good description of this fix.

Reply:
This is the merge commit...

Reply:
Whoops... 😂 SORRY!  When I opened the PR, github took me to https://github.com/firebase/firebase-android-sdk/pull/564/files/700c7c155505a388cc4f902c30e46751bbc6607a..bec9bd64fc3c8be10bb90cdf465820a3766844d4 which shows the added changelog as part of your diff...  I thought github was smarter than that. :-/